docs/proposals/multi-node/restoration.md

```diff
@@ -0,0 +1,37 @@
+# Restoration of whole cluster from the backup bucket
+A cluster may need to be restarted from backup bucket if for some reason all the persistent volumes are deleted or corrupted. This situation may occur more prominently while the cluster is hibernating during maintenance and the persistent volumes are deleted.
```

Comment:
```suggestion
This situation may occur more prominently while the cluster is hibernating during maintenance and the persistent volumes are deleted.
```

I think this statement is misleading because:
a) Gardener cluster typically don't hibernate during maintenance.
b) PVs are not deleted during hibernation.